src/pandasaurus/query.py

```diff
@@ -11,19 +11,37 @@
 
 
 class Query:
-    """Query class is responsible for returning the non-redundant graph for s subClassOf o as a simple Pandas dataframe
-    with given 2 seeds of classes, S(s) and S(o) from an initial seed, S(i)
+    """Query class is responsible for generating a pandas DataFrame that enriches the seed list with synonyms and
+    all inferred subClassOf relationships between terms in the seed. It also allows queries over the DataFrame.
+
+    Attributes:
+        enriched_df: DataFrame that is enriched with synonyms and inferred relationships between terms in
+        the seed. It will be used in further filtered queries.
+
     """
 
-    def __init__(self, seed_list: List[str]):
+    def __init__(self, seed_list: List[str], enrichment_property_list: Optional[List[str]] = None):
+        """A Query object is initialised by passing a list of seed terms (where each term is a CURIE string,
+        e.g. CL:0000001; all OBO standard CURIESs are recognised). It generates a pandas DataFrame that enriches the
+        seed list with synonyms and all inferred subClassOf relationships by default, and it supports other
+        relationships between terms in the seed. Additional methods allow enrichment with terms outside the seed
+        from slims or specified by a semantic context.
+
+        Args:
+            seed_list: A list of seed terms where each term is a CURIE string
+            enrichment_property_list: Property list to extend enrichment queries. Defaults to None
```

Comment:
In standard Python terminology, the second argument is a KWARG (keyword argument).  That it defaults to `None` is an internal implementation issue and would not normally be mentioned.